src/components/Board/DnDContextBoard.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react'
+import { DragDropContext } from 'react-beautiful-dnd'
+
+import Board, { BoardProps } from './Board'
+
+const DnDContextBoard = (props2: BoardProps) => {
```

Comment:
Why do we need a DnD ?

Reply:
Drag and drop seems good, if you want to move the boards on a list. Keep the important ones before, etc...
Seems better to try to implement it now, and remove it if we do not need it.